src/Template/Element/sentences/transcriptions.ctp

```diff
@@ -0,0 +1,28 @@
+<?php
+use App\Model\CurrentUser;
+
+if (!isset($sentenceVar)) {
+    $sentenceVar = 'vm.sentence';
+}
+
+if (CurrentUser::getSetting('show_transcriptions')) {
+    $ngIf = 'ng-if="!'.$sentenceVar.'.furigana || '.$sentenceVar.'.highlightedText"';
+} else {
+    $ngIf = 'ng-if="(vm.isMenuExpanded || !transcription.needsReview) && (!'.$sentenceVar.'.furigana || '.$sentenceVar.'.highlightedText)"';
```

Comment:
Theses strings are hard to read. In particular, it’s hard to tell which dot is used to concatenate and which dot is part of the string. I suggest you rewrite it like this:
```html
<?php
$ngIf = "!$sentenceVar.furigana || $sentenceVar.highlightedText";
?>
<div ng-if="<?= $ngIf ?>">
```